### PR TITLE
Improve survey documentation

### PR DESCRIPTION
**Pull request by marta-lokhova**

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This folder is for storing any scripts that may be helpful for using stellar-cor
         - `-sr SURVEYRESULT`, `--surveyResult SURVEYRESULT` - output file for survey results
     - sub command `analyze` - analyze an existing graph
         - `-gmla GRAPHMLANALYZE`, `--graphmlAnalyze GRAPHMLANALYZE` - input graphml file
-    - sub command `augment` - analyze an existing graph
+    - sub command `augment` - augment an existing graph with information from  stellarbeat.io. Currently, only Public Network graphs are supported.
```
Comment (bboston7) [inline scripts/README.md:22]:
```suggestion
    - sub command `augment` - augment an existing graph with information from stellarbeat.io. Currently, only Public Network graphs are supported.
```

File: docs/software/admin.md [hunk 2]
```diff
@@ -833,7 +835,24 @@ Notable field definitions
...
+    * `bytesRead`: The total number of bytes read from this peer.
+    * `bytesWritten`: The total number of bytes written to this peer.
+    * `duplicateFetchBytesRecv`: The number of bytes received that were duplicate transaction sets and quorum sets.
+    * `duplicateFetchMessageRecv`: The count of duplicate transaction sets and quorum sets received from this peer.
+    * `duplicateFloodBytesRecv`: The number of bytes received that were transactions and SCP votes duplicates.
+    * `duplicateFloodMessageRecv`: The count of duplicate transactions and SCP votes received from this peer.
+    * `messagesRead`: The total number of messages read from this peer.
+    * `messagesWritten`: The total number of messages written to this peer.
+    * `nodeId`: Node's public key.
+    * `secondsConnected`: The total number of seconds this node has been connected.
```
Comment (bboston7) [inline docs/software/admin.md:847]: Is this the number seconds the peer has been connected to this node specifically, or is it the number of seconds the peer has been connected to the network (the peer's uptime). I imagine it's the first, but it's ambiguous from the wording.

**Review by bboston7 — APPROVED** [review]
Thanks for adding more detail to the docs! I just had a clarifying question and found a small typo.

File: docs/software/admin.md (hunk 2 above)
Comment (marta-lokhova) [reply docs/software/admin.md]: Good point! It is the former. I've updated the wording to clarify this.

**Comment by graydon** [comment]
r+ ff170a1

**Pull request merged**